wiki/Beatmap/Play_time/ja.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - length
+  - playtime
+  - total length
+---
+
+# プレイ時間
+
+**プレイ時間**は、プレイしたビートマップの合計の時間です。これには、曲の開始までの時間と休憩時間も含まれます。これは通常、ビートマップの曲の長さと比例しています。
```

Comment:
missing link

for ref:

`**Play time** is the total duration of a [beatmap](/wiki/Beatmap). This includes the lead-in time and the break time(s). This usually correlates with the beatmap's song length.`